boards/arm/rpi_pico/rpi_pico_w.dts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
```

Comment:
Please write your name in the new file.

Reply:
Sorry, I'm not quite sure ow to use "rebase -i" to remove the merge commits. I re-synced the beechwoods-software fork with your repo, then ran 'git rebase -i' from a workspace where I have the branch that I'm submitting, but the merge commits are still there. Did I miss an instruction?

Is there something specific you need in the description for the PR, or do you just want some more detail for the purpose of the PR?

There is more than one non-merge commit, but they each have separate commit messages that describe the purpose of the commits, so I think everything should be good there.

Reply:
@soburi Do you have any pointers for how to remove the merge commits? A Google search is not very helpful in this regard, because a lot of the information that I see is somewhat contradictory and most of it sounds somewhat dangerous.

Reply:
> OK, I think I have the issues with the merge comments resolved and I've updated the PR description and also put my name on the one new file.

It correctly removed the merge commits. Thank you.

Commit messages still not follows guidelines.
Please confirm the following.

https://docs.zephyrproject.org/3.1.0/contribute/guidelines.html#commit-guidelines

Commit message will be following format.

```
[category]: [sub-category]: [summary]

[description...]
[description...]
[description...]

Signed-off-by: Your Name <your.email@example.com>
```

Reply:
I believe my commit now conforms to this format.